docs/operational_guide/bootstrapping.md

```diff
@@ -0,0 +1,112 @@
+# Bootstrapping
+
+## Introduction
+
+We recommend reading the [placement operational guide](placement.md) before reading the rest of this document.
+
+When an M3DB node is turned on (or experiences a placement change) it needs to go through a bootstrapping process to determine the integrity of data that it has, replay writes from the commit log, and/or stream missing data from its peers. In most cases, as long as you're running with the default and recommended bootstrapper configuration of: `filesystem,commitlog,peers,uninitialized_topology` then you should not need to worry about the bootstrapping process at all and M3DB will take care of doing the right thing such that you don't lose data and consistency guarantees are met. Note that the order of the configured bootstrappers *does* matter.
+
+Generally speaking, we recommend that operators do not modify the bootstrappers configuration, but in the rare case that you to, this document is designed to help you understand the implications of doing so.
+
+M3DB currently supports 5 different bootstrappers:
+
+1. `filesystem`
+2. `commitlog`
+3. `peers`
+4. `uninitialized_topology`
+5. `noop_all`
+
+When the bootstrapping process begins, M3DB nodes need to determine two things:
```

Comment:
slightly orthogonal - we should probably be consistent about what we been by M3DB node v host v process. Dunno if we are already but something to lookout for thru our docs